Readme.md

```diff
@@ -96,23 +129,30 @@ compile (for instance with Microsoft MSVC) you need to manually
 set/unset some switches in the compiler command line; see file *types.h*
 for a quick reference.
 
-### Resource For Understanding the Code Base
 
-* [Chess Programming Wiki](https://www.chessprogramming.org/Main_Page) 
-has good overall chess engines explanations 
-(techniques used here are well explained like hash maps etc), it was 
-also recommended by the [support team at stockfish.](http://support.stockfishchess.org/discussions/questions/1132-how-to-understand-stockfish-sources)
+## Resource For Understanding the Code Base and participating in the project
```

Comment:
Really weird title casing here. Headers for the other sections all just use normal sentence-casing, so should be the same here.

Reply:
Thanks for reading all this stuff. Not sure what you would like to read here, but I made a small modification anyway.